### ISSUE TITLE
Add properties to ModelParameter model

### ISSUE THREAD

**Issue by jonrkarr** [reproducible-biomedical-modeling/Biosimulations#411]
Add the following properties:
- target (:obj:`str`): the address of each parameter in the model (e.g., XML path). Addresses can be arbitrarily deep (i.e., parameter in reaction in submodel). This design will support all types of parameters including global kinetic parameters, local kinetic parameters to individual rate laws, and volumes of compartments.
- group (:obj:`str`): Name of the group that the parameter belongs to (e.g., 'Initial species amounts/concentrations'). Used to organize the display of parameters in the BioSimulations user interface.
- description (:obj:`str`): description
- identifiers (:obj:`list` of :obj:`Identifier`): identifiers
- type (:obj:`Type`): type of :obj:`value`
- recommendedRange (:obj:`list` of :obj:`object`): minimum and maximum recommended values of :obj:`value`

Here's an example value of `target` for parameter `k_2` of reaction `r_2`. For SBML, `target` is the path to the XML object in the XML definition of the model.
```
/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='r_2']/sbml:kineticLaw/sbml:listOfLocalParameters/sbml:localParameter[@id='k_2']/@value
```

Type is a enumeration of `boolean`, `integer`, `float`, and `string`

This also needs to be reflected into the database and REST API.

**Comment by jonrkarr** [comment]
Hold off implementing this. I think we need to make more changes to the data model to more strongly separate the details of each model format.

**Comment by bilalshaikh42** [comment]
Is this issue still open or has it been superseded by one of the others?

**Comment by jonrkarr** [comment]
Still open. I have revised the initial text.

Importing from BioModels and the tellurium image are almost done. This is driving a lot of change to the data model to accommodate things details that were hard to foresee before digging into these things. Once this is done, I don't anticipate needing to make a lot of change to the data model. I anticipate that we may need to add a little more to two specific parts of the data model
- We might need to change the models for algorithms and algorithm parameters to accommodate more simulators. I'm trying to drive that by working with a couple people to build images for iBioSim and BioNetGen. I've also tried to drive this with VCell, but I'm still waiting on simulator metadata about VCell.
- We may need to add more attributes to keep track of simulation results. I'm hoping to get to that soon now that I have a simulator image and test SED-ML files to work with.
- We may need to work through some details related to connecting simulation results to visualizations.
